docs/ref/templates/builtins.txt

```diff
@@ -2068,7 +2068,7 @@ For example:
 
 .. code-block:: html+django
 
-    {{ value|iriencode }}
+    {{ value|urlencode|iriencode }}
```

Comment:
IMO we shouldn't add `urlencode` but instead change the example below, e.g.
```
If ``value`` is ``"?test=I ♥ Django"``, the output will be ``"?test=I%20%E2%99%A5%20Django"``.
```

Reply:
I hope you don't mind that I take your example as the example for the change.